contrib/get-antlr-3.4

```diff
@@ -46,7 +46,7 @@ cd libantlr3c-3.4
 
 # Use an absolute path for the installation directory to avoid spurious libtool
 # warnings about the ANTLR library having moved
-PREFIX=$(realpath `pwd`/../..)
+PREFIX=$(python -c "import os,sys; print os.path.realpath(sys.argv[1])" `pwd`/../..)
```

Comment:
```suggestion
PREFIX=$(python -c "import os,sys; print os.path.realpath(sys.argv[1])" $(pwd)/../..)
```
Instead of mixing the two types of command substitution :)